### PR TITLE
Update direct-routing-whats-new.md

### PR DESCRIPTION
**Pull request by Venkat-Munta**
Adding the TLS changes documentation - https://learn.microsoft.com/en-us/purview/encryption-office-365-tls-certificates-changes

### CODE REVIEW

File: Teams/direct-routing-whats-new.md [hunk 1]
```diff
@@ -47,8 +47,9 @@ New TLS certificates used by Microsoft SIP interfaces will now chain up to the f
 
 Common Name of the CA: DigiCert Global Root G2
 Thumbprint (SHA1): df3c24f9bfd666761b268073fe06d1cc8d4f82a4
-The new CA certificate can be downloaded directly from DigiCert: DigiCert Global Root G2
+The new CA certificate can be downloaded directly from DigiCert: DigiCert Global Root G2.
 
+For more information, see [Office TLS Certificate Changes](https://learn.microsoft.com/en-us/purview/encryption-office-365-tls-certificates-changes)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-whats-new.md:52]:
```suggestion
For more information, see [Office TLS Certificate Changes](/purview/encryption-office-365-tls-certificates-changes)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review = @Venkat-Munta, kindly commit the suggested change. Many thanks.

**Comment by Venkat-Munta** [comment]
@JohanFreelancer9 Thanks for the correction. committed the recommended changes.

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**